src/os/ui/window/confirm.js

```diff
@@ -53,6 +53,8 @@ const launchConfirm = function(opt_options, opt_scopeOptions) {
   scopeOptions['noButtonTitle'] = options.noButtonTitle || '';
   scopeOptions['noButtonClass'] = options.noButtonClass || 'btn-secondary';
   scopeOptions['formClass'] = options.formClass || 'form-horizontal';
+  scopeOptions['checkboxText'] = options['checkboxText'] || '';
```

Comment:
Could you add both `checkbox` and `checkboxText` to `osx.window.ConfirmOptions` in our externs? Then `checkboxText` should use dot notation here, instead of bracket. That will enforce better typing in the compiler.